creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -163,18 +163,21 @@ async function _handleIssueSyncRequest({
 
   const userWallet = syncRequestParameters.data.wallet[0]
   const secondaryEndpoint = syncRequestParameters.baseURL
+  const immediate = syncRequestParameters.data.immediate
 
   const logMsgString = `_handleIssueSyncRequest() (${syncType})(${syncMode}) User ${userWallet} | Secondary: ${secondaryEndpoint}`
 
   /**
    * Remove sync from SyncRequestDeDuplicator once it moves to Active status, before processing.
    * It is ok for two identical syncs to be present in Active and Waiting, just not two in Waiting.
-   * We don't dedupe manual syncs.
    */
-  if (syncType === SyncType.Recurring) {
-    const SyncRequestDeDuplicator = require('./SyncRequestDeDuplicator')
-    SyncRequestDeDuplicator.removeSync(syncType, userWallet, secondaryEndpoint)
-  }
+  const SyncRequestDeDuplicator = require('./SyncRequestDeDuplicator')
```

Comment:
nit: this can go back to the top. I only put it here because I was trying to not import anything with shared memory for manual syncs back when we didn't need the deduplicator for manual

Reply:
moved!